airflow/operators/docker_operator.py

```diff
@@ -124,21 +136,27 @@ def __init__(
         self.volumes = volumes or []
         self.xcom_push = xcom_push
         self.xcom_all = xcom_all
+        self.dockercfg_path = dockercfg_path
+        self.registry_username = registry_username
+        self.clear_volumes = clear_volumes
 
         self.cli = None
         self.container = None
 
+        if cpus is not None:
+            logging.warning("Ignoring the cpus parameter. Use cpu_shares instead.")
```

Comment:
I am not sure it's a good idea. This fix should, Improve the integration with swarm. 
We can still support it if someone already use it. 

https://issues.apache.org/jira/plugins/servlet/mobile#issue/AIRFLOW-51